### DISCUSSION TITLE
How to use navigation sidebar instead of post metadata sidebar for blog posts

### DISCUSSION THREAD

**Discussion by sevagh** [squidfunk/mkdocs-material#6643] [Q&A] [answered]
Hello, I'm using the built-in blog plugin. I like to have the site navigation sidebar on the left, instead of the blog post metadata.

I can see the navigation bar on the blog index as expected. I copied `blog-post.html` into my overrides dir to delete the generation of the metadata left sidebar. Now as you can see in the attached screenshots, it deletes/hides all forms of left nav bar.

If I inspect the sources, I see `hidden=""` in the navigation sidebar div. If I modify this to delete the hidden keyword, the nav sidebar re-appears.

If I try to dig further, I reach a point in `base.html` where `page.meta.hide` hides the nav side bar. Are blog posts automatically setting `page.meta.hide`? https://github.com/squidfunk/mkdocs-material/blob/ab5ccbe6b8c49dbf58a25eb066a98b243a382967/src/templates/base.html#L298
![Screenshot from 2024-01-13 12-39-10](https://github.com/squidfunk/mkdocs-material/assets/8574092/630e6d67-8037-4f6d-8f1b-cc83138286b7)

![Screenshot from 2024-01-13 12-39-15](https://github.com/squidfunk/mkdocs-material/assets/8574092/fef127e5-e05a-4357-9b85-13a1634d630f)

**Comment by sevagh** [comment]
https://github.com/squidfunk/mkdocs-material/blob/ab5ccbe6b8c49dbf58a25eb066a98b243a382967/src/plugins/blog/structure/__init__.py#L124

**Comment by sevagh** [comment, marked as answer]
My workaround for now is to add this to extras.css:
```
+
+.md-sidebar--primary[hidden] {
+  display: block !important;
+}
```

**Comment by w1ndys** [comment]
this？
```
theme:
    features:
        - toc.integrate
```
